docs/src/main/asciidoc/examples.adoc

```diff
@@ -1,5 +1,22 @@
 = Examples
 
+Spring Cloud Kubernetes tries to make it transparent for your applications to consume Kubernetes Native Services
+following the Spring Cloud interfaces. For your applications, you will only need to change the implementation of these interfaces
+if you were using different DiscoveryClient or PropertySourceLocator implementations.
```

Comment:
> For your applications, you will only need to change the implementation of these interfaces
if you were using different DiscoveryClient or PropertySourceLocator implementations.

What does this mean?

Reply:
@ryanjbaxter that means that if you have for example the Eureka implementation of DiscoveryClient in your classpath you can swap that dependency for spring-cloud-kubernetes-discovery and the application should work. 
If that make sense, please help me out to express that out correctly in English :)

Reply:
I would just say that exactly what you told me, you need to add the `spring-cloud-kubernetes-discovery` dependency to your classpath and remove any other dependency that contains a `DiscoveryClient` implementation.